powerlevel9k.zsh-theme

```diff
@@ -1519,7 +1519,14 @@ $(print_icon 'MULTILINE_LAST_PROMPT_PREFIX')'
   fi
 NEWLINE='
 '
-  [[ $POWERLEVEL9K_PROMPT_ADD_NEWLINE == true ]] && PROMPT="$NEWLINE$PROMPT"
+  if [[ $POWERLEVEL9K_PROMPT_ADD_NEWLINE == true ]]; then
+    defined POWERLEVEL9K_PROMPT_ADD_NEWLINE_COUNT || POWERLEVEL9K_PROMPT_ADD_NEWLINE_COUNT=1
```

Comment:
How about a one liner...
```bash
PROMPT="$(yes '\n' | head -n$POWERLEVEL9K_PROMPT_ADD_NEWLINE_COUNT)$PROMPT"
```

Reply:
`yes` example didn't work in my OSX machine.  How about

```
if [[ $POWERLEVEL9K_PROMPT_ADD_NEWLINE == true ]]; then
  PROMPT="$(printf "\n\ufeff"%.0s {1..${POWERLEVEL9K_PROMPT_ADD_NEWLINE_COUNT:-1}})$PROMPT"
fi
```

Reply:
I tried a couple variations of the `yes` command  using `'\n'`, `'$NEWLINE'`, `'\ '`, `' '`, `''` but looks like the BSD version of `yes` doesn't play nicely.

@openam's suggestion is working for me and I'm also open to other alternatives.

Could even change it to

```zsh
[[ $POWERLEVEL9K_PROMPT_ADD_NEWLINE == true ]] && PROMPT="$(printf "\n\ufeff"%.0s {1..${POWERLEVEL9K_PROMPT_ADD_NEWLINE_COUNT:-1}})$PROMPT"
```

Reply:
That looks good to me.

Reply:
Thanks for the reviews, @openam & @Artistan!

Reply:
I've been trying to run the option I suggested for a little bit. It doesn't appear to work in all terminals. I have had issues with it in the vscode integrated terminal, and my Android JuiceSSH client. It appears that the original for loop works fine in both of them. I think it should just be included as originally proposed, unless someone else has some good ideas.